python/datashaper/datashaper/types/workflow_results.py

```diff
@@ -0,0 +1,31 @@
+"""Data-types for workflow runs."""
+import pandas as pd
+
+from dataclasses import dataclass, field
+
+
+@dataclass
+class VerbTiming:
+    """The timing of a verb in a workflow run."""
+
+    id: str
+    index: int
+    verb: str
+    timing: float
+
+
+@dataclass
+class MemoryProfile:
+    snapshot_stats: pd.DataFrame
```

Comment:
I think flake will complain about this missing docs?

Reply:
Good point, adding osme docs